recipe/meta.yaml

```
@@ -5,14 +5,14 @@ package:
   version: {{ version }}
 
 source:
-  git_url: https://github.com/cyclus/cyclus.git
-  git_rev: 1.6.0
+  git_url: https://github.com/bennibbelink/cyclus.git
+  git_rev: cleanup-dependencies
```

However, please avoid publishing from random branches or commits. You should do something like:
```suggestion
  url: https://github.com/bennibbelink/cyclus/archive/refs/tags/{{ version }}.tar.gz
  sha256: <hash>
```
and if you need, add
```
  patches:
    - patches/my_patch_on_top_of_the_tag.patch
```
Otherwise there's no traceability of the stuff that gets built in conda-forge (of course, if you control the upstream repo, you could also alter/delete tags, but those should be immutable at least in principle...)

---

We will not re-publish until we merge these changes into the upstream cyclus/cyclus repo.  Thank you for clarifying this though.

---

> We will not re-publish until we merge 

That's why I said: use the `patches:` functionality. It can be easily generated by ` git format-patch <tag>`